### DISCUSSION TITLE
For FAQ Page

### DISCUSSION THREAD

**Discussion by Schiano-NOAA** [nmfs-ost/asar#84] [Ideas]
Please use this space to add questions about basic functionality and frequently asked questions surrounding the use and intended application of asar

**Comment by Schiano-NOAA** [comment]
How should you apply asar to species complex or multispecies assessments?

**Comment by Schiano-NOAA** [comment]
Recommended practices
- double dash between ranges (2010--2021 or U.S. -- CA border)
- add \ after and between abbreviations of words (i.e. pers.\comm.\T.\Tsou) to indicate to a screen reader the proper cadence enunciation
- When using %, please add a \ before the symbol (i.e. 56\%)

**Comment by sbreitbart-NOAA** [comment]
* How to edit the template SAR citation (which may not be perfectly formatted automatically): editing text in the skeleton (right below this line: "_Please cite this publication as_").
